### ISSUE TITLE
Suggestion independent img_guideance_scale

### ISSUE THREAD

**Issue by ivaxsirc** [VectorSpaceLab/OmniGen#96]
I think that every image must be an independent img_guideance_scale

Thks

**Comment by staoxiao** [comment]
good suggestion! but this will lead to higher inference latency.

**Comment by ivaxsirc** [comment]
Right, but if you need to run tests, if you create a development branch for it i'm sure many people will want to test it. 
Thank you!
